### PR TITLE
Fix missing heading in Octernships TOS

### PR DESCRIPTION
**Pull request by ynshung**
### Why:

Missing heading in the markdown of GitHub Octernships Terms of Service for section 2.2

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added the missing heading

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@ynshung Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/site-policy/privacy-policies/github-octernships-terms-of-service.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The Program is offered to verified students on GitHub Global Campus in countries
 ## 2. Ownership and likeness
 ### 2.1 Ownership
 You hereby grant the Octernships Partners a non-exclusive, irrevocable, perpetual, royalty-free, worldwide license to make, have made, use, offer to sell, sell, import, copy, modify, create derivative works based upon, distribute, sublicense, display, perform and transmit any work product you create in connection with the Program or during your internship, including, without limitation, concepts, works, inventions, information, drawings, designs, programs, or software.  
-2.2 Likeness
+### 2.2 Likeness
```
Comment (mchammer01) [inline content/site-policy/privacy-policies/github-octernships-terms-of-service.md:24]: Just adding a newline here
```suggestion

### 2.2 Likeness
```

**Review by mchammer01 — DISMISSED** [review]
@ynshung - thanks for fixing this ✨ 
Added a small suggestion that has nothing to do with your fix. Will apply it and then merge your PR. Thanks again for helping us make our docs better 🥇

**Comment by mchammer01** [comment]
Oops, I won't be able to merge this without the approval from the @github/site-policy-admins folks 🙏🏻

**Review by khxu — APPROVED** [review]

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**